api/pkg/transformer/pipeline/compiler.go

```diff
@@ -198,10 +214,32 @@ func (c *Compiler) parseTablesSpec(tableSpecs []*spec.Table, compiledJsonPaths *
 	return NewCreateTableOp(tableSpecs), nil
 }
 
+func (c *Compiler) parseTableTransform(transformationSpecs *spec.TableTransformation, paths *jsonpath.Storage, compiledExpressions *expression.Storage) (Op, error) {
+	for _, step := range transformationSpecs.Steps {
```

Comment:
why you don't check expressions for the inputTable?

Reply:
Added the check